velox/substrait/SubstraitParser.h

```diff
@@ -86,12 +86,12 @@ class SubstraitParser {
       uint64_t id) const;
 
   /// Map the Substrait function key word into Velox function key word.
-  std::string mapToVeloxFunction(const std::string& subFunc) const;
+  std::string mapToVeloxFunction(const std::string& substraitFunction) const;
 
  private:
-  /// Used for mapping Substrait function key words into Velox functions' key
-  /// words. Key: the Substrait function key word, Value: the Velox function key
-  /// word. For those functions with different names in Substrait and Velox,
+  /// A map used for mapping Substrait function key words into Velox functions'
+  /// key words. Key: the Substrait function key word, Value: the Velox function
```

Comment:
typo: key words - keywords, key word -> keyword

What is Velox function keyword? It looks like this is just a mapping from Substrait function names to Velox function names, no?

Reply:
yes, just a mapping. I have corrected the grammar errors